src/CashuMint.ts

```diff
@@ -1,4 +1,5 @@
 import axios from "axios";
+import { Proof } from './model/Proof';
```

Comment:
because of the way the js gets transpiled, the import has to be ./model/Proof.js

check other files like CashuWallet.ts for reference